File: pgm/ABC_Cpp/ABC_419_A.cpp
#include <bits/stdc++.h>

#define __STDC_FORMAT_MACROS
#define p64 PRId64

#define FOR(i, a, b) for (int i = (a); i < (b); ++i)
#define REP(i, n) FOR (i, 0, n)
#define ALL(f, x, ...)                                                         \
    ([&] (decltype ((x)) ALL) {                                                \
        return f (begin (ALL), end (ALL), ##__VA_ARGS__);                      \
    }) (x)

using namespace std;
using ll = int64_t;

int main() {
    ios::sync_with_stdio (false);
    cin.tie (nullptr);
    string S;
    cin >> S;
    constexpr string_view red = "red";
    constexpr string_view blue = "blue";
    constexpr string_view green = "green";
    constexpr string_view SSS = "SSS";
    constexpr string_view FFF = "FFF";
    constexpr string_view MMM = "MMM";
    constexpr string_view Unknown = "Unknown";
    constexpr string_view yn = "\n";
    const unordered_map<string_view, string_view> language = {
        {red, SSS}, {blue, FFF}, {green, MMM}};
    cout << [&] {
        auto it = language.find (S);
        return it != language.end() ? it->second : Unknown;
    }() << yn;
    return 0;
}

File: pgm/ABC_R/ABC_419_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
Unknown <- "Unknown"
red <- "red"
blue <- "blue"
green <- "green"
SSS <- "SSS"
FFF <- "FFF"
MMM <- "MMM"
blank <- ""
yn <- "\n"
one <- 1L
S <- arr[one]
language <- c(
  red = SSS,
  blue = FFF,
  green = MMM
)
if (S %in% names(language)) {
  language[S] |>
    cat(yn, sep = blank)
} else {
  Unknown |>
    cat(yn, sep = blank)
}
q("no")